.github/dependabot.yml

```diff
@@ -4,5 +4,6 @@ updates:
     directory: /
     open-pull-requests-limit: 999
     rebase-strategy: disabled
+    versioning-strategy: widen
```

Comment:
This will ensure that Dependabot will bump versions both in the lockfile and the package file.